src/cycle.js

```diff
@@ -1,5 +1,6 @@
 'use strict';
 let VirtualDOM = require('virtual-dom');
+let svg = require('virtual-dom/virtual-hyperscript/svg');
```

Comment:
'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).<br>'require' is not defined.